unix/drawtext.c

```diff
@@ -79,3 +79,19 @@ void uiDrawTextLayoutExtents(uiDrawTextLayout *tl, double *width, double *height
 	*width = pangoToCairo(logical.width);
 	*height = pangoToCairo(logical.height);
 }
+
+void uiDrawLoadDefaultFont(uiFontDescriptor *f)
+{
+	GtkWidget *widget;
+	GtkStyleContext *style;
+	PangoFontDescription *fontdesc;
+
+	widget = g_object_ref_sink(gtk_drawing_area_new());
```

Comment:
A temporary GtkLabel would probably be better here (it'd behave more predictably, anyway)